scan/lib/scan/runner.rb

```diff
@@ -101,6 +93,26 @@ def handle_results(tests_exit_status)
       end
     end
 
+    def output_simulator_logs
```

Comment:
The name `include_simulator_logs` doesn't feel right. If `output` feels like the logs will be printed to the console, maybe a better name would be `copy_simulator_logs_to_output_directory` or something along those lines 😉